StereoKitC/systems/render.cpp

```diff
@@ -638,60 +642,71 @@ void render_check_screenshots() {
 		int32_t  w = render_screenshot_list[i].width;
 		int32_t  h = render_screenshot_list[i].height;
 
-		matrix view = matrix_trs(
-			render_screenshot_list[i].from,
-			quat_lookat(render_screenshot_list[i].from, render_screenshot_list[i].at));
-		matrix_inverse(view, view);
-
-		matrix proj = matrix_perspective(render_screenshot_list[i].fov_degrees, (float)w/h, render_clip_planes.x, render_clip_planes.y);
-
 		// Create the screenshot surface
-		
 		size_t   size   = sizeof(color32) * w * h;
 		color32 *buffer = (color32*)sk_malloc(size);
-		tex_t    render_capture_surface = tex_create(tex_type_image_nomips | tex_type_rendertarget);
+
+		tex_t render_capture_surface = tex_create(tex_type_image_nomips | tex_type_rendertarget);
 		tex_set_color_arr(render_capture_surface, w, h, nullptr, 1, nullptr, 8);
-		tex_release(tex_add_zbuffer(render_capture_surface));
+		tex_t render_depth_buffer = tex_add_zbuffer(render_capture_surface);
```

Comment:
This is perfectly fine and doesn't need changed, I just thought I'd note that the previous way was also correct! When adding a zbuffer to a texture, that texture holds a reference to the zbuffer internally to ensure the zbuffer's lifetime matches that of the parent texture.

Reply:
I’m glad its lifetime matches, sweet! I’ll def change it back. I also saw you noted the return of the reference is getting removed in v0.4